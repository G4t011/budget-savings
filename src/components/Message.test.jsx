import renderer from 'react-test-renderer';
import React from 'react'
import Message from './Message';

describe("Message test", () => {

    test("Should render a Message correctly", () => {
        const component = renderer.create(
            <Message type="Alert">Hello</Message>,
          );
          let tree = component.toJSON();
          expect(tree).toMatchSnapshot();
    })
})
