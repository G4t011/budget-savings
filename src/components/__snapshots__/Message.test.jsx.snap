// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message test Should render a Message correctly 1`] = `
<div
  className="alerta Alert"
>
  Hello
</div>
`;
